network-api/networkapi/buyersguide/translation.py

```diff
@@ -0,0 +1,69 @@
+from .pagemodels.products.base import Product
+from .pagemodels.products.general import GeneralProduct
+from .pagemodels.products.software import SoftwareProduct
+from .pagemodels.product_category import BuyersGuideProductCategory
+
+from modeltranslation.translator import TranslationOptions
+from modeltranslation.decorators import register
+
+from networkapi.buyersguide.pagemodels.privacy import ProductPrivacyPolicyLink
+
+
+@register(Product)
+class ProductTR(TranslationOptions):
+    fields = (
+        'name',
+        'company',
+        'blurb',
+        'worst_case',
+        'signup_requirement_explanation',
+        'personal_data_collected',
+        'biometric_data_collected',
+        'social_data_collected',
+        'how_does_it_use_data_collected',
+        'how_can_you_control_your_data',
+        'track_record_details',
+        'offline_use_description',
+        'uses_encryption_helptext',
+        'security_updates_helptext',
+        'strong_password_helptext',
+        'manage_vulnerabilities_helptext',
+        'privacy_policy_helptext',
+        'ai_helptext',
+        'price',
+        'company_track_record',
+        'phone_number',
+        'live_chat',
+        'email',
```

Comment:
Let’s drop these until we know what to actually do here ;)
Also `company_track_record` is already translated once for all via https://github.com/mozilla/foundation.mozilla.org/blob/pni-v4-2020/network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py#L33

Reply:
Sounds good!